test/kubemark/stop-kubemark.sh

```diff
@@ -55,15 +57,20 @@ if [[ "${SCALEOUT_CLUSTER:-false}" == "true" ]]; then
   do
     export TENANT_PARTITION_SEQUENCE=${tp_num}
     delete-kubemark-master
-    rm -rf "${RESOURCE_DIRECTORY}/kubeconfig.kubemark.tp-${tp_num}"
+    rm -rf "${TP_KUBECONFIG}-${tp_num}"
   done
 
   export KUBERNETES_TENANT_PARTITION=false
   export KUBERNETES_RESOURCE_PARTITION=true
   export KUBERNETES_SCALEOUT_PROXY=true
-  delete-kubemark-master
+  for (( rp_num=1; rp_num<=${SCALEOUT_RP_COUNT}; rp_num++ ))
```

Comment:
Try making delete-kubemark-master concurrent

Reply:
tracked in issue https://github.com/CentaurusInfra/arktos/issues/1036